prime-dotnet-webapi/ViewModels/Health Authority Sites/HealthAuthoritySiteSubmissionViewModel.cs

```diff
@@ -0,0 +1,27 @@
+using Prime.Models;
+
+namespace Prime.ViewModels.HealthAuthoritySites
+{
+    public class HealthAuthoritySiteSubmissionViewModel
+    {
+        public string SiteName { get; set; }
+        public PhysicalAddress SiteAddress { get; set; }
+        public string AuthorizedUserName { get; set; }
+        public string HealthAuthorityName { get; set; }
+        public string PEC { get; set; }
+        public string CareType { get; set; }
```

Comment:
should we be calling this CareSettingType?

Reply:
Same as above